flathub.json

```diff
@@ -0,0 +1,5 @@
+{
+    "skip-appstream-check": true,
+    "skip-icons-check": true,
+    "only-arches": ["x86_64"]
```

Comment:
I think that ideally it should be build from source for all architectures if possible, or at least provide repackaged binaries for x86_64 and aarch64.

Reply:
Existing haskell users on flathub are split. These two are stuck on an old unsupported runtime and only support x86: https://github.com/flathub/org.freedesktop.Bustle and  https://github.com/flathub/com.lettier.movie-monad wheras https://github.com/flathub/org.hedgewars.Hedgewars gets more by getting binaries for other architectures.